package.json

```diff
@@ -29,6 +29,7 @@
     "chromedriver": "2.46.0"
   },
   "devDependencies": {
+    "@patternfly/react-core": "^3.87.3",
```

Comment:
can you open a PR to foreman-js for it and bump the `vendor-dev` npm package? thanks

Reply:
@laviro Oops, didn't mean to keep that in there.  (We have to use PF3 for the time being :cry:)

See [discussion](https://community.theforeman.org/t/upgrading-to-patternfly-4-in-foreman/15346)